src/library_webaudio.js

```diff
@@ -37,11 +37,21 @@ let LibraryWebAudio = {
   // Wasm handle ID.
   $emscriptenGetAudioObject: (objectHandle) => EmAudio[objectHandle],
 
-  // emscripten_create_audio_context() does not itself use
+  // Performs the work of getting the AudioContext's quantum size.
+  $emscriptenGetContextQuantumSize: (contextHandle) => {
+    // TODO: in a future release this will be something like:
+    //   return EmAudio[contextHandle].renderQuantumSize || 128;
+    // It comes two caveats: it needs the hint when generating the context adding to
+    // emscripten_create_audio_context(), and altering the quantum requires a secure
+    // context and fallback implementing. Until then we simply use the 1.0 API value:
+    return 128;
```

Comment:
Couldn't this code already today read

```js
return EmAudio[contextHandle].renderQuantumSize || 128;
```

That way the code would also be verifying that `contextHandle` is a valid context.

Reply:
It could, but I was worried about its `[SecureContext]` marking in the spec and what that entails, and also since we can't yet set it then it doesn't make a difference.

But I'll happily change it and look at it again if/when it breaks, and when the API allows for the setting.

Reply:
Ok, works either way.

[SecureContext] just means that the parameter won't be populated if the page is served via an insecure http:// handler. The value won't then be present, and will return `undefined`.

(http://localhost URLs get a special case and is always considered secure).

Reply:
My concern was it'd throw an exception, not just silently return undefined.

(For localhost it still needs `Cross-Origin-Opener-Policy` and `Cross-Origin-Embedder-Policy` headers for audio demos to run, BTW, since `SharedArrayBuffer` isn't available and throws a reference error)

Reply:
> My concern was it'd throw an exception, not just silently return undefined.

Ah, right. An exception won't happen, that is guaranteed by the IDL spec of [SecureContext]: https://www.w3.org/TR/secure-contexts/#integration-idl which states multiple times

![image](https://github.com/user-attachments/assets/9e630d72-7816-4f84-a990-38b1d77b8189)

But let's leave this later for when an implementation comes along.